### PR TITLE
Sync Changes

### PR DESCRIPTION
**Pull request by Dank-del**

### CODE REVIEW

File: packages/ui/components/ui/tag.tsx [hunk 1]
```diff
@@ -1,18 +1,28 @@
...
+const Tag = React.forwardRef<HTMLButtonElement, Props>(
+    ({ className, ...props }, ref) => {
+        return (
+            // <div className="">
+            //     <div className={cn(`"flex flex-row items-center justify-center p-1"`, props.selected && "bg-[#007BFF]")}>
+            //         <div className="relative leading-[16px] font-medium">{props.text}</div>
+            //     </div>
+
+            // </div>
+            <button {...props} className={cn("relative rounded-[100px] overflow-hidden shrink-0 flex flex-row items-center justify-center py-1 px-3 box-border text-center text-[12px] text-[#F0F7FF]", props.selected ? "bg-[#007BFF]" : "bg-[#141414]")}>
```
Comment (dhruvsol) [inline packages/ui/components/ui/tag.tsx:18]: lets not use colors directly here, update the config and then use them

File: packages/ui/components/ui/project-card.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                                {props.matched && <div className="flex flex-row items-center justify-start gap-[3px]">
+                                    <div className="relative leading-[20px] font-medium">{props.matched}</div>
+                                    <div className="relative leading-[20px]">match</div>
+                                </div>}
+                            </div>
+                        </div>
+                    </div>
+                    <div className="flex flex-col items-start justify-start py-3 px-0 text-[14px]">
+                        <div className="rounded-md bg-[#00000000] box-border w-[114px] h-11 overflow-hidden shrink-0 flex flex-row items-center justify-center py-0 px-4 gap-[4px] border-[2px] border-solid border-[rgba(138, 138, 138, 0.25)]">
+                            <svg className="relative w-[18px] h-[18px] overflow-hidden shrink-0" viewBox="0 0 24 24">
```
Comment (dhruvsol) [inline packages/ui/components/ui/project-card.tsx:55]: change w-[18px] to something like w-4.5 and update tailwind config

File: apps/cubik-web/src/app/components/explorer-page/Projects.tsx [hunk 3]
```diff
@@ -72,29 +68,52 @@ const ProjectTabs = () => {
...
                                 <p className="mt-2">{project.description}</p>
                             </div>
                         </div>
+                        <Button onClick={() => toast(`Saved ${project.name}`, {
+                            style: {
+                                background: '#000',
+                                color: '#fff',
+                            }
+                        })} className='gap-2 rounded-sm' variant="outline">
+                            <svg className="relative w-[18px] h-[18px] overflow-hidden shrink-0" viewBox="0 0 24 24">
```
Comment (dhruvsol) [inline apps/cubik-web/src/app/components/explorer-page/Projects.tsx:106]: lets not use svgs directly add them in icon lib or use `<Image/>` tag

File: apps/cubik-web/src/app/components/explorer-page/Projects.tsx [hunk 4]
```diff
@@ -72,29 +68,52 @@ const ProjectTabs = () => {
 
     return (
         <div className="w-full sm:w-2/3 mx-auto mt-8">
-            <Subhead/>
-            <div className="flex space-x-2 mt-2">
+            <Subhead text='Projects' endElement={
```
Comment (dhruvsol) [inline apps/cubik-web/src/app/components/explorer-page/Projects.tsx:71]: Change svg to Image or icon lib

**Review by dhruvsol — CHANGES_REQUESTED** [review]

File: packages/ui/components/ui/tag.tsx (hunk 1 above)
Comment (Dank-del) [reply packages/ui/components/ui/tag.tsx:18]: Irfan told me to hardcode the colors for now

Comment (dhruvsol) [reply packages/ui/components/ui/tag.tsx:18]: I will talk to him for now and change it to come from config @Dank-del

File: apps/cubik-web/src/app/components/explorer-page/Projects.tsx (hunk 4 above)
Comment (Dank-del) [reply apps/cubik-web/src/app/components/explorer-page/Projects.tsx]: Made it into an svg component

**Pull request merged**